### ISSUE TITLE
Non-binary websocket messages

### ISSUE THREAD

**Issue by dbrgn** [saltyrtc/saltyrtc-meta#137]
What should we do when non-binary websocket messages arrive?

Right now I think they're simply dropped in all implementations. But maybe we should raise a ProtocolError instead?

Does the spec already say something about this case?

**Comment by lgrahl** [comment]
I don't think it does specify what needs to happen. Sure, as always, let's raise `ProtocolError` in such a case.
